src/liquid/place/bulk_create_form.html

```diff
@@ -110,6 +111,10 @@
             fileName.textContent = fileInput.files[0].name;
           }
         }
+        const form = document.querySelector("form#form_place_create");
+        form.addEventListener("GotoList", function(e) {
```

Comment:
can you explain why you're doing this?

Reply:
I am doing this so i can swap the `Add` view with the progress bar before initiating the redirect to the product list. For this i trigger a custom event that signals when we're ready to redirect. Before, the redirect happened immediately but since the place list takes some time to load, the bulk upload form will still be visible until the request for the the place list page completes

Reply:
Wonder if chunking the table response would be another option?  So rather than waiting for the whole table to be ready and the whole response to come in, we could send some minimal chunk.

In my mind, this might be a little better because it avoids the complexity of this custom event and the swapping complexity + it is a good practice for large pages

Reply:
Sounds good, we can definitely show the table then lazy load the data

Reply:
I've implemented the change, first we load the skeleton table then load the items.